docs/basic-strict.md

```diff
@@ -0,0 +1,77 @@
+# Basic strict property typing
+
+With a configured bootstrap, we use annotations on the properties to make your app operate in strict mode.
+
+The `StrictPhp` uses compatible with the PhpDocumentor annotation. Then you probably should be familiar with them.
```

Comment:
"`StrictPhp` uses PhpDocumentor compatible annotations" (link the annotations doc of phpdocumentor, please).